app/default_overrides_test.go

```diff
@@ -62,7 +62,7 @@ func TestDefaultAppConfig(t *testing.T) {
 
 	assert.Equal(t, uint64(1500), cfg.StateSync.SnapshotInterval)
 	assert.Equal(t, uint32(2), cfg.StateSync.SnapshotKeepRecent)
-	assert.Equal(t, "0.1utia", cfg.MinGasPrices)
+	assert.Equal(t, "0.002utia", cfg.MinGasPrices)
```

Comment:
[question] how was `.002utia` arrived at? IIUC `0.1utia` was arrived at based on rough estimates. How did the estimates change to go from 0.1 -> 0.01 -> 0.002?

Reply:
If the original value was off by two orders of magnitude, I would expect 0.1 -> 0.01 -> 0.001

Reply:
Since `.002utia` got merged here, I think we may want to update chain-registry to 

```
        "low_gas_price": 0.002,
        "average_gas_price": 0.004,
        "high_gas_price": 0.008
```

but we likely only want to do this after a decent chunk of validators have updated to a release that includes this. On second thought, I think their locally configured default won't get overridden with this new default so our release notes should probably explicitly call out the manual action that validators need to take.

Reply:
https://github.com/cosmos/chain-registry/pull/3239
https://github.com/chainapsis/keplr-chain-registry/pull/290

updating these now, but leaving in draft